models/banner.js

```diff
@@ -0,0 +1,68 @@
+'use strict';
+
+const Joi = require('joi');
+const mutate = require('../lib/mutate');
+
+const MODEL = {
+    id: Joi
+        .number()
+        .integer()
+        .positive(),
+
+    message: Joi
```

Comment:
Can we add a field to record the time when this created? like this https://github.com/screwdriver-cd/data-schema/blob/913538c6de39a0c6edb2fa1f2068e1766d81f711/models/build.js#L78

Reply:
That will be helpful